Task - 1/PrimeNumber.java

```diff
@@ -0,0 +1,92 @@
+
+public class PrimeNumber {
+	public static void main(String[] args) {
+		primeNumbers(1, 100);
+		checkNumberType(496); // Perfect Number
+		checkNumberType(12321); // Palindrome Number
+		checkNumberType(1634); // Armstrong Number
+		checkNumberType(123); // None of these
+
+	}
+
+	public static boolean isPrime(int number) {
+		int count = 0;
+		if (number >= 2) {
+			for (int i = 2; i <= number; i++) {
+				if ((number % i) == 0) {
+					count++;
+				}
+			}
+		}
+
+		if (count == 1) {
+			return true;
```

Comment:
Can you try optimizing the solution? Few points to note,
Instead of checking all the numbers (2<=i<=number), you can exit the loop whenever the considered number is divisible by any number less than the considered number.
Also, there's a way to optimize the number of iterations. Can you explore that as well?